processes/third_party_vuln_process.md

```diff
@@ -77,7 +77,12 @@ At this point, a CVE should be requested through the HackerOne platform through
 [email](cve-assign@hackerone.com) that should include the Report ID and a summary.
 
 The vulnerability is considered as published when a Pull Request adding it
-to this repository is opened.
+to this repository is opened. Once the Pull Request is merged, and a new version
+is released, vulnerable versions must be deprecated using `npm deprecate`:
```

Comment:
_must_ is a bit aggressive since this isn't something we can force on maintainers.

I'd also rephrase maybe to convey that this note about deprecation isn't very helpful for us as triage members but rather something we need to convey to the maintainers, for example through one of:
1. open an issue in their repo to offer them to run this
2. tell them on the hackerone report already since they're already there as part of the discussion and the fixed release